snoop/html.py

```diff
@@ -30,32 +30,29 @@ def _extract_links(html):
     return html
 
 def _create_lxml_html_cleaner():
+    # Consult http://lxml.de/3.4/api/lxml.html.clean.Cleaner-class.html
     cleaner = lxml.html.clean.Cleaner()
 
-    # Only the tags will be removed, their content will get pulled up into
-    # the parent tag. The idea here is to make it explicit to copy/paste a link
-    # Also remove any tags that would make inline printing bad (html, head, body)
+    # The idea here is to make it explicit to copy/paste a link, as all of our
+    # links will be available as a text-only index.
+    # Also remove any tags that would make inline printing bad, like <html>,
```

Comment:
All of those are being removed, but by setting the `cleaner.links` and `cleaner.style` props.